serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-grpc-services.adoc

```diff
@@ -1 +1,164 @@
-= Orchestration of gRPC based services
\ No newline at end of file
+= Orchestration of gRPC based services
+
+Learn how to invoke a gRPC service from a Kogito Serverless Workflow running with Quarkus. This guide covers:
+
+* Creating a workflow that invokes a running gRPC service. 
+* Configuring properties to locate the running service. 
+
+== Prerequisites
+
+It's expected that you have read the how to link:create-your-first-workflow-service.html[Create Your First Workflow] guide. 
```

Comment:
```suggestion
It's expected that you have read the how to xref:getting-started/create-your-first-workflow-service.adoc[Create Your First Workflow] guide.
[Create Your First Workflow] guide. 
```
When referencing to other page in same module we need to use cross-reference `xref:` to `folder-name/page-name` containing the `adoc` file.
Refer [Antora docs](https://docs.antora.org/antora/2.0/navigation/link-syntax-and-content/)